pkg/feeds/sequence/sequence.go

```diff
@@ -0,0 +1,221 @@
+package sequence
+
+import (
+	"context"
+	"encoding/binary"
+	"errors"
+
+	"github.com/ethersphere/bee/pkg/crypto"
+	"github.com/ethersphere/bee/pkg/feeds"
+	"github.com/ethersphere/bee/pkg/storage"
+	"github.com/ethersphere/bee/pkg/swarm"
+)
+
+type index struct {
+	index uint64
+}
+
+func (i *index) MarshalBinary() ([]byte, error) {
+	indexBytes := make([]byte, 8)
+	binary.BigEndian.PutUint64(indexBytes, i.index)
```

Comment:
in different places in the codebase we use different endianness, we should probably converge to a unified approach here